app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,305 @@
+@import "font-awesome";
+
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+body {
+  /* background-color: #f5f6f8; */
+  background-image: linear-gradient(#538252, #64c674);
+  font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+  color: #888;
+}
+
+a {
+  color: #000;
```

Comment:
- [ ] please make sure to use the color scheme provided in the template:
![image](https://github.com/SAZZAD1Q2/BudgetMate/assets/39240820/1792d09e-e2df-4795-9f1d-3238c2861eed)